.github/workflows/cicd-terraform-auth0.yml

```diff
@@ -14,6 +14,7 @@ on:
     branches: ["main"]
 
 env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think GITHUB_TOKEN is a default secret which is repository scoped, therefore don't think you need to create an env var for it here.

Reply:
I think it's something to do with the add-pr-comment action - see my message in the chat [here](https://mojdt.slack.com/archives/CPVD6398C/p1722507720095929)